### ISSUE TITLE
LaTeX raises error for labeled code-block

### ISSUE THREAD

**Issue by tk0miya** [sphinx-doc/sphinx#2523]
Since 66fdb340f7f82ee57709f754eb732a24e9607bc5 (for #2507), following reST code causes an error.

```
Welcome to test's documentation!
================================

Contents:

.. _label:

::

   some code
```

Error message is here:

```
(/USR/LOCAL/TEXLIVE/2015/texmf-dist/tex/latex/amsfonts/umsb.fd) [1{/USR/LOCAL/T
EXLIVE/2015/texmf-var/fonts/map/pdftex/updmap/pdftex.map}] [2]
Adding blank page after the table of contents.
[1] [2]

! LaTeX Error: Something's wrong--perhaps a missing \item.

See the LaTeX manual or LaTeX Companion for explanation.
Type  H <return>  for immediate help.
 ...

l.146 \begin{Verbatim}[commandchars=\\\{\}]

?
```

@jfbu It seems this is a side effect of 66fdb340f7f82ee57709f754eb732a24e9607bc5. Could you check this please?

**Comment by jfbu** [comment]
Sorry about that. The `\addvspace` raises such a mysterious error when it is used in a paragraph. This was the case here because the produced `.tex` file had no blank line before the start of the `Verbatim` environment. Wil be fixed by PR #2524

**Comment by jfbu** [comment]
Fixed at e093846dccc778a903fb59740c48fcd0789316d4

**Comment by tk0miya** [comment]
confirmed! Thank you for quick response :-)
